app/src/main/java/com/scv/slackgo/activities/LocationActivity.java

```diff
@@ -58,6 +59,50 @@ protected void onCreate(Bundle savedInstanceState) {
     protected void onStart() {
```

Comment:
all this was moved to onCreate, I was recommended to change onStart for onResume